_infra/docker/Dockerfile

```diff
@@ -1,8 +1,10 @@
 FROM eclipse-temurin:21-jre-alpine
 
+COPY ./cacert.pem /usr/local/share/ca-certificates/cacert.pem
+RUN update-ca-certificates
```

Comment:
You'll get TLS x509 errors if you don't add the Umbrella cert to the image being built locally on the corporate Mac. We won't be doing this going forward, this is just for this POC hack.